src/Core/Checkout/Cart/Delivery/Struct/DeliveryCollection.php

```diff
@@ -26,7 +26,7 @@ public function sortDeliveries(): self
                 return -1;
             }
 
-            return $a->getDeliveryDate()->getEarliest() > $b->getDeliveryDate()->getEarliest();
+            return (int) ($a->getDeliveryDate()->getEarliest() > $b->getDeliveryDate()->getEarliest());
```

Comment:
```suggestion
            return ($a->getDeliveryDate()->getEarliest() <=> $b->getDeliveryDate()->getEarliest());
```

should probably used here: see "Spaceship" operator https://www.php.net/manual/en/language.operators.comparison.php

definitely needs a test, if this is the same behaviour as before

Reply:
Actually this seems to be done by purpose, as a location check is done a few lines earlier yielding a `-1` - I would rather not mess with that in this PR